docs/use-cases/401k.md

```diff
@@ -7,9 +7,209 @@ parent: Use Cases
+* others will just fail due to auto-enroll preferences. 
+* Some companies will enroll everyone and then near the end of the fiscal year (for us is always with calendar year QBO) just notify the owner of the plan of missing compliance data (SSN or Tax ID)
+
+*Access to employee SSN and date of bith is not included by default, work with your Intuit partner to get the necessary scopes associated with your app.*
+
+After eestablishing connecton with QBO company run this  query for employee roster
+
+```GRAPHQL
+query readEmployeeRoster {
+  company {
+		id
+		
+    industryType
+    companyType
+    employees{
+      nodes {
+        id
+        firstName
+        lastName
+        displayName
+        
+        payrollProfile {
+          preferredPhone
+          preferredAddress {
+            streetAddress1
+            state
+          }
+          compensations {
+            id
+            active
+            amount
+          }
+          deductions {
+            id
+            amount {
+              percentage
+              value
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+#### Contribution Level
+
+401(k) Providers will have their own screens for sign up and enrollment of employees of the company, but will require that information be transferred back to QB Payroll so that the correct deductions can be taken. 
+ 
+They will then need to write (and read) these fields for contribution per employee level. The only person who can edit the names or fields of each contribution will be the creator. If you are taking over from a different company as 401(K) provider work with QBO company's payroll admins to sunset contributions from previous plan. 
+
+* Traditional % or Traditional $
+* Roth % or Roth $
+* loan amount $ 
+* Loan ID
+* Employer contribution %
+* Employer contributions limit
+* Nonelective contribution 
+* Deduction Code
+
+```
+ADD MUTATOIN SAMPLE
+```
+
+#### Paycheck Information
+
+401k providers need to download payslips after each payroll run so they know how much in deductions to take out for the employees, and this is where deductions details are most important.   At the beginning of the plan, providers will need to query for up to all of last fiscal year’s payslips to determine plan eligibility for each employee:
+
+* Pay date
+* Payroll Process Date (approved date)
+* Employee ID
+* SSN (again above entitlements)
+* Paycheck ID
+* Gross Pay
+* YTD Pay (optional, not launch blocker)
+* Payroll Type (optional - bonus, overtime, regular, etc)
+* Hours Worked
+* Amount deducted for each category (employee)
+  * Roth
+  * Traditional
+  * Loan
+* Employer contribution  
+
+After eestablishing connecton with QBO company run this  query to download payslips periodically.
+
+```GRAPHQL
+query readAllEmployeePayslips {
+  company {
+	id
+    employees{
+      nodes {
+        id
+        }
+        payslips {
```

Comment:
This query doesn't look quite right to me, the brace (`}`) closes before `payslips`.  Could you double check this is the right query?